interface/README.md

```diff
@@ -0,0 +1,51 @@
+# LXD Interface
```

Comment:
As discussed we don't want the `interface` directory but just the provider side and that directly in `src/interfaces.py` similar to how other operator charms are doing it.